indexers/keyvalue/RedisDBIndexer/README.md

```diff
@@ -1,3 +1,45 @@
 # RedisDBIndexer
 
-Use Redis as a key-value indexer 
\ No newline at end of file
+Key-value indexer wrapper around the Redis database.  
```

Comment:
Some link about Redis? Some explanation about it being in-memory and stuff like this would be good maybe?

Reply:
Good point. How about now?